src/pages/verify/getting-started/oidc-intro.mdx

```diff
@@ -75,8 +75,8 @@ If you want to receive the response in another way you must specify the `respons
 | `acr_values`    | Identifies which eID identity service you want to use. You can only specify one value, and it must identify the exact type of identity service, as some countries have, for example, both a mobile and web based service. Possible values can be found in the [authorize request builder](/verify/guides/authorize-url-builder/#auth-methods--acr-values). |
 | `response_mode` | (optional) Specifies how you want your result delivered via the `redirect_uri`: Use `query` to return the `code`/`id_token` as a query parameter, `fragment` to have it delivered on a URL fragment, and finally `form_post` to have it posted back to your `redirect_uri`.  <br />  <br /> Default values are `query` for `response_type=code` and `query` for `response_type=id_token`. |
 | `state`         | (optional but recommended) An opaque arbitrary alphanumeric string your app adds to the initial request that Criipto includes when redirecting back to your application. |
-| `login_hint` | (optional) Various use cases leverage this parameter. You can read more about them [here](#the-loginhint-parameter).|
-
+| `login_hint`    | (optional) Various use cases leverage this parameter. You can read more about them [here](#the-loginhint-parameter).|
+| `prompt`        | (optional) Specifies whether the user will be forced to re-authenticate. Possible values are `none` (default) and `login`. More information is available [here](/verify/guides/sso/#per-request-options). |
```

Comment:
`prompt=none` is only the default if SSO is enabled I believe. We also support the non-standard parameters `consent` and `consent_revoke`

Reply:
Maybe:

Specifies whether the user will be forced to re-authenticate. Possible values are `none`, `login`, `consent`, and `consent_revoke`. More information is available [here](/verify/guides/sso/#per-request-options) and in our [authorize URL builder](/verify/guides/authorize-url-builder/).

Reply:
> Maybe:
> 
> Specifies whether the user will be forced to re-authenticate. Possible values are `none`, `login`, `consent`, and `consent_revoke`. More information is available [here](/verify/guides/sso/#per-request-options) and in our [authorize URL builder](/verify/guides/authorize-url-builder/).

Sounds good, if we can avoid "here" and replace with "in our SSO guide" i'd be great.